### ISSUE TITLE
visIgraph can't find imported layout function

### ISSUE THREAD

**Issue by rbagd** [datastorm-open/visNetwork#222]
I think this snippet in `R/visIgraph.R` should be either removed or adapted to search in additional environments:

```
if(length(findFunction(layout)) == 0){
    stop("Can't find '", layout, "' function. Please verify it")
}
```
As an example, if you are importing `visIgraph` into your own custom package with
```
#' @importFrom visNetwork visIgraph
```
for some function `f()` then you'll keep getting the above error (if you're using the defaults) as long as you don't specifically load `igraph` library prior to running `f()`. Importing `layout_nicely` into the package namespace doesn't help either and `layout` function remains unfound.

**Comment by bthieurmel** [comment]
Hi,

I'll try to fix that this week for next version.

**Comment by bthieurmel** [comment]
fix on ``2.0.2``, just submitted on CRAN
